zhaquirks/tuya/ts011f_plug.py

```diff
@@ -95,6 +95,68 @@ class Plug(EnchantedDevice):
     }
 
 
+class CircuitBreaker(EnchantedDevice):
+    """Tuya smart circuit breaker with metering (_TZ3000_qeuvnohg). https://www.aliexpress.com/item/1005004747066832.html """
+
+    signature = {
+        MODEL: "TS011F",
+        ENDPOINTS: {
+            # <SimpleDescriptor endpoint=1 profile=260 device_type=266
+            # device_version=1
+            # input_clusters=[0, 3, 4, 5, 6, 1026, 1794, 2820, 57344, 57345]
+            # output_clusters=[10, 25]>
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_PLUG_IN_UNIT,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    0x0402,
+                    Metering.cluster_id,
+                    ElectricalMeasurement.cluster_id,
+                    TuyaZBE000Cluster.cluster_id,
+                    TuyaZBExternalSwitchTypeCluster.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [Time.cluster_id, Ota.cluster_id],
+            },
+            # <SimpleDescriptor endpoint=242 profile=41440 device_type=97
+            # device_version=1
+            # input_clusters=[]
+            # output_clusters=[33]>
+            242: {
+                PROFILE_ID: 41440,
+                DEVICE_TYPE: 97,
+                INPUT_CLUSTERS: [],
+                OUTPUT_CLUSTERS: [GreenPowerProxy.cluster_id],
```

Comment:
Dont deleting the Zigbee GP endpoint in the replacement is one part of the device function.

Reply:
Yes, I will keep the 242 endpoint, what confused me was that the "Plug" class does not keep it.

Reply:
I am not sure, what is the meaning of the 0x402 cluster, it reports as "temperature" but the value is constant and always "-630" so it has likely some other meaning...

Reply:
@zleba I confirm it on my batch (all 4 show this, and it's certainly not a temperature), I think you are on the right path to ignore this reading, it's also not in the manual (as you confirmed [here](https://github.com/zigpy/zha-device-handlers/issues/2005) )